subtree-splits/blt-project/composer.json

```diff
@@ -73,7 +73,11 @@
         "patchLevel": {
             "drupal/core": "-p2"
         },
-        "patches": []
+        "patches": {
+            "drupal/memcache": {
+                "12987574 - >1M data writes incompatible with memcached (memcached -I 32M -m32): #435694": "https://www.drupal.org/files/issues/2019-02-26/435694-118-d8-memcache-large-items.patch"
```

Comment:
👍 🎉

Reply:
Ideally, to get the memcache issue merged, we should provide before/after stack metrics over the course of a few days. The one reason why memcache maintainers would be reluctant to merge this is if there would be a negative performance impact. Memcache is too critical in people's infrastructures to introduce performance regressions. If we could demonstrate that it doesn't have any, then I'm sure we'd get more support.